CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ option(TRITON_ENABLE_GPU "Enable GPU support in libraries" OFF)
 
 set(TRITON_COMMON_REPO_TAG "main" CACHE STRING "Tag for triton-inference-server/common repo")
 set(TRITON_THIRD_PARTY_REPO_TAG "main" CACHE STRING "Tag for triton-inference-server/third_party repo")
+set(TRITON_CORE_REPO_TAG "main" CACHE STRING "Tag for triton-inference-server/core repo")
```

Comment:
This is needed only for the C API right?
@deadeyegoodwin @tanmayv25 we should have build flags to enable/disable the C API, TF Serving and torchserve support. So that users can customize the client library build if they require.
@jbkyang-nvi Could you also file a ticket for this?

Reply:
We already have the ticket. JIRA is under-maintenance hence unable to pull up the exact ticket.